docs/Cli.md

```diff
@@ -246,6 +246,8 @@ Re-apply any new defaults as desired.
 |  failsafe_fw_roll_angle  | -200 | Amount of banking when `SET-THR` failsafe is active on a fixed-wing machine. In 1/10 deg (deci-degrees). Negative values = left roll |
 |  failsafe_fw_pitch_angle  | 100 | Amount of dive/climb when `SET-THR` failsafe is active on a fixed-wing machine. In 1/10 deg (deci-degrees). Negative values = climb |
 |  failsafe_fw_yaw_rate  | -45 | Requested yaw rate to execute when `SET-THR` failsafe is active on a fixed-wing machine. In deg/s. Negative values = left turn |
+|  failsafe_min_distance  | 0 | If failsafe happens when craft is below this distance in meters from home, failsafe will not execure regular failsafe_procedure, but will execute procedure specified in failsafe_min_distance_procedure instead. 0 = Normal failsafe_procedure always taken. |
```

Comment:
1. Probably "closer than" is a better word than "below".
2. "execure" - a typo, should be "execute"

Reply:
Thanks. done.